server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -112,6 +113,14 @@ export class RestGitService {
     }
 
     public async getRef(ref: string): Promise<git.IRef> {
+        if (this.writeToExternalStorage)
+        {
+            const getRefParams: IGetRefParamsExternal = {
+                config: { enabled: true },
+            };
+            const params = encodeURIComponent(JSON.stringify(getRefParams));
+            return this.get(`/repos/${this.getRepoPath()}/git/refs/${ref}/${params}`);
```

Comment:
See previous comment on the route handler, but I think a uri-encoded-stringified-JSON should be a query param (i.e. `?config=${params}`)